tests-ng/test_azure_times.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+from plugins.systemd import Systemd
+
+@pytest.mark.booted(reason="Test runs systemd")
+@pytest.mark.root(reason="To start the systemd service")
+@pytest.mark.modify(reason="Starts systemd service")
+@pytest.mark.performance_metric
+@pytest.mark.feature("Azure instance to test boot time for kernel, initrd and userspace") 
```

Comment:
check other example of the `feature` marker for reference. It should contain a boolean expression, not a prose sentence. You can provide a reason using the reason argument, that is optional.

Reply:
this is the reason you are getting this error in ci 

> INTERNALERROR> boolean.boolean.ParseError: Invalid symbols sequence such as (A B) for token: "instance" at position: 13


We have a library in place to evaluate if a test should be run based on which features are in the image. there are examples for how to use that in other tests.